app/src/main/res/layout/fragment_detailed_item.xml

```diff
@@ -24,6 +24,13 @@
             android:textSize="30sp"
             tools:text="Title" />
 
+        <ImageView
+            android:id="@+id/edit_item_image"
```

Comment:
Maybe change the id so it's not the same as in the edit fragment, or it would make sense to change the id so it doesn't contain edit.

Reply:
Yeah, that's a good point!